packages/session-replay-browser/src/config/joined-config.ts

```diff
@@ -22,41 +23,88 @@ export class SessionReplayJoinedConfigGenerator {
       const samplingConfig =
-        this.remoteConfigFetch &&
-        ((await this.remoteConfigFetch.getRemoteConfig(
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+        (await this.remoteConfigFetch.getRemoteConfig(
           'sessionReplay',
           'sr_sampling_config',
           sessionId,
-        )) as SessionReplayRemoteConfig['sr_sampling_config']);
+        )) as SessionReplayRemoteConfig['sr_sampling_config'];
 
-      if (samplingConfig && Object.keys(samplingConfig).length > 0) {
-        if (Object.prototype.hasOwnProperty.call(samplingConfig, 'capture_enabled')) {
-          config.captureEnabled = samplingConfig.capture_enabled;
-        } else {
-          config.captureEnabled = false;
-        }
+      const privacyConfig =
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+        (await this.remoteConfigFetch.getRemoteConfig(
+          'sessionReplay',
+          'sr_privacy_config',
+          sessionId,
+        )) as SessionReplayRemoteConfig['sr_privacy_config'];
 
-        if (samplingConfig.sample_rate) {
-          config.sampleRate = samplingConfig.sample_rate;
+      if (samplingConfig || privacyConfig) {
+        remoteConfig = {};
+        if (samplingConfig) {
+          remoteConfig.sr_sampling_config = samplingConfig;
+        }
+        if (privacyConfig) {
+          remoteConfig.sr_privacy_config = privacyConfig;
         }
-      } else {
-        // If config API response was valid (ie 200), but no config returned, assume that
-        // customer has not yet set up config, and use sample rate from SDK options,
-        // allowing for immediate replay capture
-        config.captureEnabled = true;
       }
     } catch (err: unknown) {
       const knownError = err as Error;
       this.localConfig.loggerProvider.warn(knownError.message);
       config.captureEnabled = true;
     }
 
+    if (!remoteConfig) {
+      return config;
+    }
+
+    const { sr_sampling_config: samplingConfig, sr_privacy_config: privacyConfig } = remoteConfig;
+    if (samplingConfig && Object.keys(samplingConfig).length > 0) {
+      if (Object.prototype.hasOwnProperty.call(samplingConfig, 'capture_enabled')) {
+        config.captureEnabled = samplingConfig.capture_enabled;
+      } else {
+        config.captureEnabled = false;
+      }
+
+      if (samplingConfig.sample_rate) {
+        config.sampleRate = samplingConfig.sample_rate;
+      }
+    } else {
+      // If config API response was valid (ie 200), but no config returned, assume that
+      // customer has not yet set up config, and use sample rate from SDK options,
+      // allowing for immediate replay capture
+      config.captureEnabled = true;
+      this.localConfig.loggerProvider.debug(
+        'assuming config API response was valid, but no config returned. session replay capture enabled.',
```

Comment:
Nit this log ends up being what the customer sees if they have their log level set high enough, so maybe something a little more customer friendly like:
`Remote config successfully fetched, but no values set for project, Session Replay capture enabled`